src/lib.rs

```diff
@@ -260,6 +260,7 @@ pub use os::OsRng;
 
 pub use isaac::{IsaacRng, Isaac64Rng};
 pub use chacha::ChaChaRng;
+pub use sample::{sample, sample_reservoir};
```

Comment:
I wonder if we should remove this pub use? Especially because I want to introduce a [Sample](https://dhardy.github.io/rand/rand/trait.Sample.html) trait.